client/src/styles/_button.scss

```diff
@@ -124,3 +124,18 @@
 .icon-mail:before {
   border-radius: 0.1rem;
 }
+
+// WHO ARE OUR USERS SURVEY BUTTON
+.woau-survey-button {
```

Comment:
Think this is a typo - should be `waou`, right?

Reply:
oops yes, thank you!